src/main/groovy/org/mockito/release/notes/vcs/DefaultCommitApprover.java

```diff
@@ -0,0 +1,31 @@
+package org.mockito.release.notes.vcs;
+
+import org.mockito.release.notes.model.Commit;
+import org.mockito.release.notes.util.Predicate;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+public class DefaultCommitApprover implements Predicate<Commit> {
```

Comment:
But should then all skip `messages` be postfix? Earlier `[ci skip]` could be in any index of commit message.